jsonrpc-specs/src/methods/discv5.json

```diff
@@ -0,0 +1,567 @@
+			"name": "routingTableInfoResult",
+			"description": "Discv5 routing table information",
+			"required": true,
+			"schema": {
+				"title": "routingTableInfoResults",
+				"description": "Routing table details",
+				"type": "object",
+				"required": [
+					"localKey",
+					"buckets"
+				],
+				"properties": {
+					"localKey": {
+						"title": "nodeId",
+						"description": "The key identifying the local peer that owns the routing table.",
+						"$ref": "#/components/schemas/bytes32"
+					},
+					"buckets":  {
+						"title": "kBucketsTable",
+						"description": "Represents a Kademlia routing table.",
+						"$ref": "#/components/schemas/kBuckets"
+					}
+				}
+			}
+		}
+	},
+	{
+		"name": "discv5_setEnr",
+		"summary": "Write an ethereum node record to the database.",
+		"params": [
+			{
+				"name": "enr",
+				"required": true,
+				"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+				}
+			}
+		],
+		"result": {
+			"name": "setEnrResult",
+			"description": "Returns ethereum node record if the node record has been successfully saved.",
+			"schema": {
+				"title": "Ethereum node record",
+				"$ref": "#/components/schemas/Enr"
+			}
+		}
+	},
+	{
+		"name": "discv5_getEnr",
+		"summary": "Fetch the latest ENR associated with the given node ID",
+		"params": [
+			{
+				"name": "node_id",
+				"required": true,
+				"schema": {
+					"title": "NodeId",
+					"$ref": "#/components/schemas/bytes32"
+				}
+			}
+		],
+		"result": {
+			"name": "getEnrResult",
+			"description": "Returns latest ENR associated with the given node ID.",
+			"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+				      }
+		}
+	},
+	{
+		"name": "discv5_deleteEnr",
+		"summary": "Delete a Node ID from the local database",
+		"params": [
+			{
+				"name": "node_id",
+				"required": true,
+				"schema": {
+					"title": "NodeId",
+					"$ref": "#/components/schemas/bytes32"
+				}
+			}
+		],
+		"result": {
+			"name": "deleteEnrResult",
+			"description": "Returns None upon successful deletion of the node record.",
+			"schema": {
+				"type": "string",
+				"pattern": "None$"
+			}
+		}
+	},
+	{
+		"name": "discv5_lookupEnr",
+		"summary": "Fetch the ENR representation associated with the given Node ID and optional sequence number",
+		"params": [
+			{
+				"name": "node_id",
+				"required": true,
+				"schema": {
+					"title": "NodeId",
+					"$ref": "#/components/schemas/bytes32"
+				}
+			},
+			{
+				"name": "sequence_number",
+				"schema": {
+					"type": "number"
+				}
+			}
+		],
+		"result": {
+			"name": "lookupEnrResult",
+			"description": "Returns ENR associated with the given node ID",
+			"schema": {
+					"title": "Ethereum node record",
+					"$ref": "#/components/schemas/Enr"
+			}
+		}
+	},
+	{
+		"name": "discv5_sendPing",
+		"summary": "Send a PING message to the specified node",
+		"params": [
+			{
+				"name": "node_id",
```

Comment:
This is called `node_id` but we actually need to provide an ENR. Change name to `enr`? (same for other messages below)